react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx

```diff
@@ -0,0 +1,269 @@
+import React from "react";
+import { useHoverInfo } from "../../components/Map";
+import DeckGLMap from "../../DeckGLMap";
+import InfoCard from "../../components/InfoCard";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Slider } from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+import { ContinuousLegend } from "@emerson-eps/color-tables";
+
+export default {
+    component: DeckGLMap,
+    title: "DeckGLMap / Map Float Layer",
+} as ComponentMeta<typeof DeckGLMap>;
+
+type NumberQuad = [number, number, number, number];
+type Views = {
+    layout: [number, number];
+    viewports: [
+        {
+            id: string;
+            show3D: boolean;
+        }
+    ];
+};
+
+const valueRange = [-3071, 41048];
+
+// Example using "Map" layer. Uses float32 mesh and properties binary arrays. Not PNG.
+const meshMapLayer = {
+    "@@type": "MapLayer",
+    id: "mesh-layer",
+    meshUrl: "hugin_depth_5_m.float32",
+    frame: {
+        origin: [432150, 6475800],
+        count: [1451, 1141],
+        increment: [5, 5],
+        rotDeg: 30,
+    },
+    propertiesUrl: "kh_netmap_5_m.float32",
+    contours: [0, 100.0],
+    isContoursDepth: true,
+    material: true,
+    colorMapName: "Physics",
+};
+
+const defaultArgs = {
+    bounds: [432150, 6475800, 439400, 6481500] as NumberQuad,
+};
+
+const defaultParameters = {
+    docs: {
+        inlineStories: false,
+        iframeHeight: 500,
+    },
+};
+
+function gradientColorMap(x: number) {
+    return [255 - x * 255, 255 - x * 100, 255 * x];
+}
+
+function nearestColorMap(x: number) {
+    if (x > 0.5) return [100, 255, 255];
+    else if (x > 0.1) return [255, 100, 255];
+    return [255, 255, 100];
+}
+
+function breakpointColorMap(x: number, breakpoint: number) {
+    if (x > breakpoint) return [0, 50, 200];
+    return [255, 255, 0];
+}
+
+function createColorMap(breakpoint: number) {
+    return (value: number) => breakpointColorMap(value, breakpoint);
+}
+
+const axes_hugin = {
+    "@@type": "AxesLayer",
+    id: "axes-layer2",
+    bounds: [432150, 6475800, -3500, 439400, 6481500, 0],
+};
+const north_arrow_layer = {
+    "@@type": "NorthArrow3DLayer",
+    id: "north-arrow-layer",
+};
+export const MapLayer: ComponentStory<typeof DeckGLMap> = () => {
```

Comment:
I think the bounds need to be adjusted in this story: 

![Screenshot from 2022-07-04 07-13-12](https://user-images.githubusercontent.com/19621822/177085766-61a022df-d954-4395-bbe0-7c92cab8cbda.png)